Classes/Domain/Model/Company.php

```diff
@@ -505,11 +441,16 @@ public function setInstagram(string $instagram): void
      * SF: Do not add PoiCollection as strict_type to $txMaps2Uid
      * as this will break DataMap in Extbase when maps2 is not installed.
      */
-    public function getTxMaps2Uid()
+    public function getTxMaps2Uid(): ?PoiCollection
```

Comment:
PLease...read the comment above ;-)

Reply:
And yes: I see your hasTyMaps2Uid method, but it was not called in DataMapper::thawProperties